databricks/koalas/frame.py

```diff
@@ -25,6 +25,7 @@
 
 import numpy as np
 import pandas as pd
+import pyspark.sql.functions as F
```

Comment:
nit: an unnecessary change?

Reply:
Yes, I didn't notice the import on a different line!